tests/test_fernet.py

```diff
@@ -25,10 +26,18 @@ def json_parametrize(keys, filename):
     vector_file = cryptography_vectors.open_vector_file(
         os.path.join("fernet", filename), "r"
     )
+
+    def _id_gen():
+        # generate readable test identifiers
+        for idx in itertools.count():
+            yield f"{filename}:{next(idx)}"
```

Comment:
```suggestion
            yield f"{filename}[{idx}]"
```

`next` isn't needed since we're using a `for` loop, and using `[]` makes it clear it's not a line number -- `:` is often used for lines.